### PR TITLE
Spacing standardization

### PR DESCRIPTION
**Pull request by roninopf**

### CODE REVIEW

**Comment by vbabenkoru** [comment]
@roninopf Hmm if we reformat everything on `inbox`, it will be a lot of pain to merge _anything_ from master to it since it'd conflict on every file... I think it might be better to reformat on `master` and then merge into `inbox`... That'll produce a lot of conflicts in that merge, but at least that'll be a one-time thing.

**Comment by roninopf** [comment]
> @roninopf Hmm if we reformat everything on `inbox`, it will be a lot of pain to merge _anything_ from master to it since it'd conflict on every file... I think it might be better to reformat on `master` and then merge into `inbox`... That'll produce a lot of conflicts in that merge, but at least that'll be a one-time thing.

Shouldn't it be fine since `inbox` is up to date with `master`?

**Comment by vbabenkoru** [comment]
Conflicts happen when the same line (or an adjacent line) is changed in both branches; if `inbox` has whitespace changes that don't exist on `master`, and you modify these or adjacent lines in `master` and then try to merge, that'll trigger merge conflicts.

**Comment by roninopf** [comment]
I think I'm still confused, because theoretically these changes are some of the last to be put in, so by that point, all the changes currently in `inbox` would be merged into `master` before this PR.

File: swift-sdk/Internal/Operations+Extensions.swift [hunk 1]
```diff
@@ -1,50 +0,0 @@
-//
-//  Operations+Extensions.swift
-//  swift-sdk
-//
-//
-
-import Foundation
-
-class AsyncOperation: Operation {
```
Comment (vbabenkoru) [inline swift-sdk/Internal/Operations+Extensions.swift]: Is this not used anywhere?

File: Tests/swift-sdk-swift-tests/IterableAPITests.swift [hunk 2]
```diff
@@ -589,24 +590,20 @@ class IterableAPITests: XCTestCase {
         let config = IterableConfig()
         IterableAPI.initializeForTesting(apiKey: IterableAPITests.apiKey, config: config, networkSession: networkSession)
         IterableAPI.email = "user@example.com"
-        IterableAPI.get(
```
Comment (vbabenkoru) [inline Tests/swift-sdk-swift-tests/IterableAPITests.swift]: Why only this method? :) Looks like formatting is inconsistent in the other methods too.

File: Tests/swift-sdk-swift-tests/IterableAPITests.swift [hunk 3]
```diff
@@ -589,24 +590,20 @@ class IterableAPITests: XCTestCase {
...
+                            let expectedQueryParams = [(name: AnyHashable.ITBL_KEY_API_KEY, value: IterableAPITests.apiKey),
+                                                       (name: AnyHashable.ITBL_KEY_COUNT, value: 1.description),
+                                                       (name: AnyHashable.ITBL_KEY_PLATFORM, value: .ITBL_PLATFORM_IOS),
+                                                       (name: AnyHashable.ITBL_KEY_SDK_VERSION, value: IterableAPI.sdkVersion)]
+                            TestUtils.validate(request: networkSession.request!,
+                                               requestType: .get,
+                                               apiEndPoint: .ITBL_ENDPOINT_API,
+                                               path: .ITBL_PATH_GET_INAPP_MESSAGES,
+                                               queryParams: expectedQueryParams)
+                            expectation1.fulfill()},
```
Comment (vbabenkoru) [inline Tests/swift-sdk-swift-tests/IterableAPITests.swift]: nit: `}` isn't very visible here, I think placing the closing bracket on a new line looks better.

File: swift-sdk/Internal/Operations+Extensions.swift (hunk 1 above)
Comment (roninopf) [reply swift-sdk/Internal/Operations+Extensions.swift]: Correct!

**Pull request merged**